www/components/shelf/shelf.js

```diff
@@ -25,6 +28,25 @@ class shelf extends LitElement {
     this.expandRoute(window.location.pathname);
   }
 
+  async setupShelf(page) {
+    console.log('setupShelf for page =>', page);
```

Comment:
yeah, we'll definitely want to catch.  I did notice that in prod mode (at least) the shelf seems to load twice, even from the graph.  I understand that it would load once for no route, then when the route is set, but it seems to do that twice?  Something I'm keeping my eye on and will show on our next call.  👀